src/renderer/components/layout/setting-layout.scss

```diff
@@ -34,7 +34,7 @@
 
     .sidebar {
       width: 218px;
-      padding: 60px 0 60px 20px;
+      padding: 60px 10px 60px 20px;
```

Comment:
Might sound small, but we should start moving to direction of [8pt-grid](https://medium.com/swlh/the-comprehensive-8pt-grid-guide-aa16ff402179)